src/pages/index.js

```diff
@@ -31,6 +31,12 @@ const features = [
           </b>{" "}
           Explains how to develop the user interface of Home Assistant.
         </p>
+        <p>
+          <b>
+            <a href="docs/android">Android</a>.
+          </b>{" "}
+          Explains how to contribute to the Android companion apps.
```

Comment:
```suggestion
          Explains how to contribute to the Android companion app.
```

Reply:
Well we do have different apps. Wear and App are different, but I'll modify it.